assets/xml/objects/object_okuta.xml

```diff
@@ -20,7 +20,7 @@
         <Texture Name="gOctorokFadeToBlackTex" OutName="fade_to_black" Format="rgb5a1" Width="8" Height="8" Offset="0x30D0"/>
 
         <!-- Octorok Skeleton -->
-        <Skeleton Name="gOctorokSkel" Offset="0x3660"/>
+        <Skeleton Name="gOctorokSkel" Type="Normal" LimbType="Standard" Offset="0x3660"/>
```

Comment:
Since you've corrected this one, could you correct the Skeleton Types listed as `Standard` to `Normal` as well? There's 6 of them:
```
assets/xml/objects/object_dekunuts.xml
assets/xml/objects/object_dnk.xml
assets/xml/objects/object_dns.xml
assets/xml/objects/object_dodongo.xml
assets/xml/objects/object_firefly.xml
assets/xml/objects/object_hintnuts.xml```

Reply:
Why do we have 6 listed as `Standard`? It seems weird to me. I think there should be a reason for this :thinking:

Reply:
people are mixing up Standard Limbs and "normal" skeletons. I guess ZAP just defaults to normal for skeleton if the name is unrecognized

Reply:
ah this is why lol 
`else if (skelTypeStr != "Normal" && skelTypeStr != "Standard")`
ideally limbs will have a type called Standard, and skeleton is something else. I mentioned in discord possibly just making flex and skin attributes of the skeleton so we arent forced to give the base skeleton a name. still need to think it over some more though

for now I would just make sure all skeletons use Normal instead of Standard

Reply:
I found the culprit. 
```py

def skeleton_to_xml(var_name,offset, type):
    skel_type = "Flex" if "Flex" in type else "Standard"
    limb_type = "Standard"

    return skeleton_xml.format(var_name, offset.lstrip('0'), skel_type, limb_type)
```
It was `tools/xmlcreate.py`.